### PR TITLE
Add packages required for arch

### PR DESCRIPTION
**Pull request by mariogrip**

### CODE REVIEW

File: Porting.md [hunk 1]
```diff
@@ -4,9 +4,7 @@
 
 Setting up your environment is necessary before starting the porting process.
 
-### Using Ubuntu
-
-#### Step 1: Installing required packages
+#### Step 1: Installing required packages (Ubuntu/debian)
```
Comment (z3ntu) [inline Porting.md]: Debian uppercase?

File: Porting.md [hunk 2]
```diff
@@ -17,6 +15,38 @@ sudo apt-get install git gnupg flex bison gperf build-essential \
...
+
+First install this GPG key (it's needed for a PGP signature in the ncurses5-compat-libs and lib32-ncurses5-compat-libs packages)
+
+```
+gpg --recv-keys 702353E0F7E48EDB
+```
+
+Then install the lineageos-devel meta package from AUR
+
+**With yaourt**
```
Comment (z3ntu) [inline Porting.md]: Please don't include yaourt - it has so many problems... pacaur is the most popular one right now and it works a lot better.

**Comment by vanyasem** [comment]
I rebased `master` into `mariogrip/arch` and fixed all formatting issues. Can we merge this now?

**Comment by vanyasem** [comment]
Merged `master` into `mariogrip/arch` and fixed all issues. Can we merge this now?

**Comment by vanyasem** [comment]
closing as a #60 was megerd

**Pull request closed without merging**